packages/gafl-webapp-service/src/pages/contact/contact/__tests__/route.spec.js

```diff
@@ -79,35 +79,44 @@ describe('name > route', () => {
       }
     )
 
-    it.each([
-      ['Email', '12M', 'Email test@email.com', 'test@email.com'],
-      ['Text', '12M', 'Email', undefined],
-      ['Email', '8D', 'Email', 'test@email.com'],
-      ['Text', '8D', 'Email', undefined]
-    ])(
-      'when preferredMethodOfConfirmation is %s and licenceLength is %s, emailText returns %s',
-      async (preferredMethodOfConfirmation, licenceLength, expected, email) => {
-        const licensee = { birthDate: 'birthDate', preferredMethodOfConfirmation, email }
-
-        const result = await getData(getMockRequest({ licensee, licenceLength }))
-        expect(result.emailText).toBe(expected)
-      }
-    )
+    it.each`
+      desc                                                                                                      | preferredMethodOfConfirmation | licenceLength | junior   | email                  | expected
+      ${'Email confirmation, 12 month licence, not a junior licence so emailText returns Email test@email.com'} | ${'Email'}                    | ${'12M'}      | ${false} | ${'test@email.com'}    | ${'Email test@email.com'}
+      ${'Text confirmation, 12 month licence, not a junior licence so emailText returns Email'}                 | ${'Text'}                     | ${'12M'}      | ${false} | ${undefined}           | ${'Email'}
+      ${'No confirmation, no licence length, junior licence so emailText returns Email'}                        | ${undefined}                  | ${'12M'}      | ${true}  | ${undefined}           | ${'Email'}
+      ${'Email confirmation, 8 day licence, not a junior licence so emailText returns Email'}                   | ${'Email'}                    | ${'8D'}       | ${false} | ${'example@email.com'} | ${'Email'}
+      ${'Text confirmation, 8 day licence, not a junior licence so emailText returns Email'}                    | ${'Text'}                     | ${'8D'}       | ${false} | ${undefined}           | ${'Email'}
+    `('$desc', async ({ preferredMethodOfConfirmation, licenceLength, junior, email, expected }) => {
```

Comment:
I think you've now made the description unnecessarily complicated. There was no problem with the parameterised description before, and now there's duplication between the description and parameters. Please could you leave the description as it was, with the various parameters fed in?